tests/test_sign_cmd.py

```diff
@@ -32,7 +32,7 @@ def test_sign_tx_change_old_protocol(cmd, public_key_bytes):
         )
         for x in range(5)
     ]
-    change_index = fake.pyint(0, 5)
+    change_index = fake.pyint(0, 4)
```

Comment:
`fake.pyint` generates from 0 to 5 inclusive, this had a 1/6 chance of failing if we chose the 5 as an index.